jws.go

```diff
@@ -155,6 +159,11 @@ func (parsed *rawJsonWebSignature) sanitized() (*JsonWebSignature, error) {
 			}
 		}
 
+		// Check that there is not a nonce in the unprotected header
+		if sig.Header != nil && sig.Header.Nonce != "" {
+			return nil, fmt.Errorf("square/go-jose: Nonce parameter included in unprotected header")
```

Comment:
Since this error gets reused everywhere, I would prefer if it was defined in a (private) constant. That should also make it easier to match on the specific error for testing.